.github/workflows/release.yaml

```diff
@@ -62,6 +69,15 @@ jobs:
             --path="./deploy/kustomize" \
             --source="${{ github.event.repository.html_url }}" \
             --revision="${GITHUB_REF_NAME}/${GITHUB_SHA}"
+      - name: update-kustomize-assert
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          kustomize build deploy/kustomize/base > kustomize-base.yaml
```

Comment:
The kustomize base don't contain any rbac at all, so it's probably not worth uploading it. My guess is that users have no use for it, so I see no reason to upload it.